### PR TITLE
Remove codedom code

### PR DESCRIPTION
**Pull request by Forgind**
CodeDOM should always be on.

### CODE REVIEW

**Comment by Forgind** [comment]
Resolves #937.

File: src/Build.UnitTests/BackEnd/ItemCreationTask.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.Build.Framework;
```
Comment (rainersigwald) [inline src/Build.UnitTests/BackEnd/ItemCreationTask.cs]: Can you add the copyright header to the new files?

```
// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

File: src/Build.UnitTests/BackEnd/ItemCreationTask.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```
Comment (rainersigwald) [inline src/Build.UnitTests/BackEnd/ItemCreationTask.cs]: One blank line after this, please

/tinynit

**Pull request merged**